source/adminguide/templates.rst

```diff
@@ -272,6 +273,13 @@ To upload a template:
       the Featured Templates list. Only an administrator can make a
       template Featured.
 
+Uploading Templates from local computer
+-------------------------------------------
+
+It's also possible to upload already prepared template from your local computer.
+Steps are similar as when Uploading template from remote HTTP server, except that you need to choose local template file from your PC.
+For this feature to work, your SSVMs must be supporting HTTPS (for more info please visit `“Using a SSL Certificate for the Console Proxy” 
```

Comment:
Are you sure that this is true?
SSVM is a client not a server in this instance.
There is a bug regarding remote https not being recogniased
https://github.com/apache/cloudstack/issues/3172

Reply:
https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=39620237 says "The Apache web server will only accept upload requests over HTTPS" and yes, tested this (for other reasons) and found it's NOT working when your SSVM doesn't have SSL configured (captured logs confirming HTTPS url "was not found" kind of thing - that is reason I opened this PR.
Confirmed it works if SSVM is SSL enabled.